starter/assets-base/browserconfig.xml

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should be in assets-base of the starter, and also provided by default (along with manifest.json, favicon and other icons) in stark-ui